bytecode/bytecode_5e938f0.cpp

```diff
@@ -297,7 +297,7 @@ Error GDScriptDecomp_5e938f0::decompile_buffer(Vector<uint8_t> p_buffer) {
 				line += "self";
 			} break;
 			case TK_BUILT_IN_TYPE: {
-				line += Variant::get_type_name(Variant::Type(tokens[i] >> TOKEN_BITS));
+				line += get_type_name_v3(tokens[i] >> TOKEN_BITS);
```

Comment:
Some old `3.0 dev` versions should use Godot 2 Variant:

5e938f0 / 2017-02-28
c24c739 / 2017-01-20
f8a7c46 / 2017-01-11
62273e5 / 2017-01-08
8b912d1 / 2017-01-08
23381a5 / 2016-12-17
513c026 / 2016-10-03
4ee82a2 / 2016-08-27
1add52b / 2016-08-19

As well as all early `4.0 dev` versions use version 3 Variant:

f3f05dc / 2020-02-13
506df14 / 2020-02-12